web/pandas/about/governance.md

```diff
@@ -49,145 +49,110 @@ manage project donations and acts as a parent legal entity. NumFOCUS is the
+consensus from the rest of activate maintainers
+
+See the list of active maintainers [here](team.html#maintainers).
+
+### BDFL (Benevolent Dictator For Life)
+
+The figure of the BDFL exist to be able to unblock situations where a decision
+needs to be made, and consensus or voting has failed. In such situations, the
+BDFL will make the final decision.
+
+**Membership**: Wes McKinney, as original creator of pandas has been the BDFL
+of the project. In the event of Wes stepping down as BDFL, maintainers will
+make a decision about whether to appoint a new BDFL or change the governance
+type.
+
+### Finances committee
+
+The role of the members of the finances committee is to approve the spending
+of pandas funds. Decisions in general will be made together with the rest of
+active maintainers. Committee members will be responsible to make the final
+decisions, and formally approve payments.
+
+**Membership**: The committee will have 5 members, who will be selected by active
+maintainers. Some constraints exists regarding committee membership:
+
+- Members must be active maintainers
+- No more than two committee members can be employed directly or indirectly
+  by the same employer
+- Committee members should not have conflicts of interest that could prevent
+  them to make the best decisions in the interest of the project. This includes
+  maintainers who receive significant payments from pandas funds
+
+### Code of conduct committee
+
+The role of the committee is to make sure pandas is as open, transparent and
+inclusive as it aims to be by its values. In particular, the committee will
+monitor and respond to any possible violation of our code of conduct. And
+will publish regular summaries about violation reports.
+
+**Membership**: Any members of the community can be part of the committee.
+The committee will have 5 members, who will be selected by active maintainers.
+The next constraints must be satisfied:
+
+- The committee should aim to be as diverse as reasonably possible, to be able
+  to make decisions based on a variety of points of views. In particular, the
+  committee should not have more than 3 members of the same gender, or more
+  than two members from the same geography (continent). Ideally the committee
+  will also be diverse in other ways such as religion, political views,
+  age, etc.
+- No more than two members of the committee should be pandas maintainers.
+
+### Inactive maintainers
+
+Inactive maintainers are former active maintainers who are not interested or
+not available to continue contributing to pandas in a regular way. If they
+decide to participate in a discussion, they will still be considered active
+maintainers for that discussion, but otherwise they are not expected to be part
+of the decision making of the project, not have commit rights to the pandas
+repositories, or be in the maintainers distribution list.
+
+**Membership**: Active maintainers become inactive by their own decision.
```

Comment:
My first proposal for this membership condition:

1. An active maintainer that voluntarily notifies the active maintainers to be inactive by their own decision
2. An active maintainer that satisfies all of the following:

- Has not been engaged on Github, mailing lists, or any other project communication channel for over 1 year
- Has not responded to an inquiry from the active maintainers, within a 1 month time frame, that they want to remain an active maintainer.

IMO I think having just active/inactive (with whatever title we choose) with a broad condition is sufficient.

Reply:
I'm fine with this proposal.  I like condition (2), because it then defines a process for the core team to mark people as "inactive", as opposed to waiting for an active person to declare they are inactive.

It also covers the unfortunate case of someone passing away, and the only way we find out is due to the fact that they are non-responsive.